test/parsing-test.js

```diff
@@ -31,13 +34,24 @@ describe('Messages parsing', function () {
 
     afterEach(function () {
         room.destroy()
+        createIssue.restore()
         createMilestone.restore()
         closeMilestone.restore()
         sendMsg.restore()
         sendGst.restore()
         getRoom.restore()
     })
-    
+    it('create issue', function () {
+        return room.user.say('mikanebu', `bot issue "This is test issue" about "We want to test" in "datahq/docs""`).then(function () {
+            assert.equal(createIssue.callCount, 1)
+            assert.equal(room.messages[1][1].substr(0, 15), "@mikanebu Issue")
+        })
+    })
+    it('create issue with typo', function () {
```

Comment:
I think this test is not useful

Reply:
@anuveyatsu It is working, but, yes, it is not super useful. I will remove it